charts/primary-site/templates/cronjobs/garbage-collector.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: garbage-collector
+  labels:
+    app: foxglove-garbage-collector
+spec:
+  schedule: "*/10 * * * *" # every 10 minutes
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: 1
+  successfulJobsHistoryLimit: 3
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          volumes:
+            - name: cloud-credentials
+              secret:
+                secretName: gcp-cloud-credential
+                optional: true
+          containers:
+            - name: garbage-collector
+              image: us-central1-docker.pkg.dev/foxglove-images/images/garbage-collector:dcb5b4812e4d0a6054638ea43712f2e7ee5050ab
+              volumeMounts:
+                - mountPath: /secrets
+                  name: cloud-credentials
+              envFrom:
+                - secretRef:
+                    name: cloud-credentials
+                    optional: true
+              env:
+                - name: GOOGLE_APPLICATION_CREDENTIALS
+                  value: /secrets/credential.json
+                - name: FOXGLOVE_API_URL
+                  value: "{{ .Values.globals.foxglove_api_url }}"
+                - name: FOXGLOVE_SITE_TOKEN
+                  valueFrom:
+                    secretKeyRef:
+                      name: foxglove-site
```

Comment:
I think the name of the secret should be "garbage-collector" and the key should be "foxgloveSiteToken"

Reply:
or if we want to use one secret, maybe something like "foxglove", "dataPlatform", "primarySite". I think calling this "foxglove-site" is not too great because we are making the user care about the word "site" without giving them the benefit of being able to disambiguate between types of site

Reply:
We should use one secret.

Reply:
The site token will be the same between garbage-collector and inbox-listener though, so we might as well use the same `foxglove-site` key for both?